cookbooks/aws-parallelcluster-slurm/templates/default/slurm/login_nodes_daemon_service/login_nodes_daemon_script.sh.erb

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
```

Comment:
I suggest to execute this script with errexit and xtrace mode, i.e. adding `set -ex` right after the shebang.
With the errexit mode the script is gonna exit immeditelyt with failure if an inner command (pipelined or compiund) fails.

With the xtrace mode is like debugging mode for bash scripts: the script will log the executed commands together with their outputs.

Reply:
Have added it.